src/responsibleai/rai_analyse/_rai_insight_data.py

```diff
@@ -0,0 +1,475 @@
+from responsibleai import RAIInsights
+import json
+
+import os
+import pandas as pd
+import sklearn.metrics as skm
+
+# from erroranalysis._internal.constants import Metrics
+from erroranalysis._internal.metrics import metric_to_func
+from fairlearn.metrics import selection_rate
+
+from datetime import datetime
+# from raiwidgets.cohort import ClassificationOutcomes, Cohort, CohortFilter, CohortFilterMethods
+
+from collections import OrderedDict
+
+
+def false_positive(y_test, y_pred):
+    tn, fp, fn, tp = skm.confusion_matrix(y_test, y_pred).ravel()
+    return fp
+
+
+def false_negative(y_test, y_pred):
+    tn, fp, fn, tp = skm.confusion_matrix(y_test, y_pred).ravel()
+    return fn
+
+
+metric_func_map = metric_to_func
+metric_func_map["error_rate"] = skm.zero_one_loss
+metric_func_map["selection_rate"] = selection_rate
+metric_func_map["confusion_matrix"] = skm.confusion_matrix
+metric_func_map["false_positive"] = false_positive
+metric_func_map["false_negative"] = false_negative
+
+
+class ExplainerFiles:
+    GLOBAL_IMPORTANCE = "global_importance_values.json"
+    LOCAL_IMPORTANCE = "local_importance_values.json"
+    FEATURES = "features.json"
+    CLASSES = "classes.json"
+
+
+def get_metric(metric, y_pred, y_test):
+    return metric_func_map[metric](y_pred, y_test)
+
+
+class RaiInsightData:
+    def __init__(self, raiinsight_path):
+        os_path = os.path.join(*os.path.split(raiinsight_path))
```

Comment:
I'm guessing that this is about making sure there's an absolute path?

Reply:
it converts the input to proper os path separator, for example, ./raiinsight becomes .\\raiinsight on windows